Production_scripts/Process_extracts/Produce_extract_tests/extract_district_nursing_tests.R

```diff
@@ -0,0 +1,43 @@
+####################################################
+# Name of file - extract_district_nursing_tests.R
+# Original Authors - Zihao Li
+# Original Date - October 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for district nursing:
+#               district nursing processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+
+# Read in Data-----------------------------------------
+
+year <- check_year_format("1920")
+
+# Read new data file
+# new_data <- readr::read_rds(get_source_extract_path(year, "DN"))
+new_data <-
+  haven::read_spss("/conf/sourcedev/Source_Linkage_File_Updates/1920/DN_for_source-201920.zsav") %>%
+  dplyr::rename(year = Year)
```

Comment:
I think this rename is only required because you read in the SPSS file? It should be removed, everything in the `rds` files should be `snake_case` and if it isn't we need to fix it there.

@Jennit07 can we look at 'fixing' this in the SPSS too please? SPSS doesn't care about case so we can just swap `Year` for `year` without it affecting anything in the SPSS code.